modules/onepassword-to-secret-manager/main.tf

```diff
@@ -0,0 +1,21 @@
+locals {
+  value = { for k, item in var.data : item.secret_key => module.onepass.pass[k] }
+}
+
+
+module "onepass" {
+  source = "dasmeta/shared/any//modules/onepassword"
```

Comment:
the current module misses some common files like providers.tf, version/requirements.tf?

can we have the cloud where the secret getting created be used in modules names like
onepassword-to-aws-secret-manager
keeper-to-gcp-secret-manager

?